buddypress/common/js-templates/messages/parts/bp-messages-form.php

```diff
@@ -53,13 +53,13 @@ class="send-to-input"
 	<div id="bp-message-content"></div>
 
 	<?php
-	if ( 'mac' === $os ) {
+	if ( 'window' === $os ) {
 		?>
-		<p class="bp-message-content_foot_note"><span class="space_note">Please note:</span><span class="space_note"><strong><?php esc_html_e( 'Shift+Return', 'buddyboss' ); ?> </strong> <?php esc_html_e( 'to add a new line', 'buddyboss' ); ?></span><strong><?php esc_html_e( 'Return', 'buddyboss' ); ?></strong><?php esc_html_e( ' to Send', 'buddyboss' ); ?></p>
+		<p class="bp-message-content_foot_note"><span class="space_note">Please note:</span><span class="space_note"><strong><?php esc_html_e( 'Shift+Enter', 'buddyboss' ); ?> </strong> <?php esc_html_e( 'to add a new line', 'buddyboss' ); ?></span><strong><?php esc_html_e( 'Enter', 'buddyboss' ); ?></strong><?php esc_html_e( ' to Send', 'buddyboss' ); ?></p>
 		<?php
-	} elseif ( 'window' === $os ) {
+	} else {
```

Comment:
It looks like this change makes it so Linux will now use the "return" terminology… Is that desired? I would have thought they would be using "enter".

Reply:
You may well be right. I'm hoping San can fill us in. Easy to fix once we know.